guides/common/modules/proc_viewing-multiple-content-view-environment-hosts.adoc

```diff
@@ -0,0 +1,47 @@
+[id="Viewing_multiple_content_view_environment_hosts{context}"]
+= Viewing multiple content view environment hosts
+
+Use this procedure to view multiple content view environment activation keys in Hammer CLI
+
+.Prerequisites
+Register a host with the created activation key. 
+For more information, see xref:Using_Activation_Keys_for_Host_Registration_{context}[Using activation keys for host registration].
+
+[NOTE]
+====
+Click the *Insecure* checkbox when registering a host.
+====
+
+.Procedure
+. In the {ProjectwebUI}, navigate to *Hosts* > *All Hosts*.
+. Click the *New UI* button.
+. Click *Manage columns*.
+. Under *Content*, click the *Content view environments* checkbox and click *Save*.
+. The multiple content view environments for the host are displayed under the *Content view environments* column.
+. Click on the content view environments to display details.
+. Click on the vertical ellipsis to the right of the content view environments and select *Change content view environments” from the drop-down menu.
+
+[WARNING]
+====
+This host is associated with multiple content view environments. 
+If you assign a lifecycle environment and content view here, the host will be removed from the other environments.
+====
+
+On the *Hosts Details* page, the *Content view environments* card displays the details of the content view environments. 
+
+.CLI procedure
+. Run the following command to view the host:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer host list --organization “Default Organization”
+----
+This lists multiple content view environments for the host.
++
+. Run the following command to view host information:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer host info –_”Information_ID”_
+----
+The information for the host is displayed including multiple content view environments.
```

Comment:
Same comments as about the activation key procedure above:
- call out the columns
- fix –_”Information_ID”_